src/gsp.py

```diff
@@ -228,6 +230,8 @@ def get_truths_for_a_specific_gsp(
     #### Parameters
     - **gsp_id**: _gsp_id_ of the requested forecast
     - **regime**: can choose __in-day__ or __day-after__
+    - **start_datetime_utc**: optional start datetime for the query.
+    If not set, defaults to N_HISTORY_DAYS env var, which if not set defaults to yesterday.
```

Comment:
It could be helpful to say what format the `start_datetime_utc` should be in. ex. 2023-07-22...

Reply:
I think its a `datetime` so you dont have to specific the formant